apps/src/lib/script-editor/ScriptEditor.jsx

```diff
@@ -394,6 +395,22 @@ export default class ScriptEditor extends React.Component {
               visible={!this.props.hidden}
               pilotExperiment={this.props.pilotExperiment}
             />
+            <label>
+              Is a CourseOffering
+              <input
+                name="is_course"
+                type="checkbox"
+                defaultChecked={this.props.isCourse}
+                style={styles.checkbox}
+              />
+              <HelpTip>
+                <p>
+                  (Still in development) If checked, indicates that this Unit
+                  represents a standalone CourseOffering. Examples of such Units
+                  include CourseA-F.
```

Comment:
Can we call this a "standalone course" in the UI instead of "CourseOffering"? (any input @dmcavoy ?) I don't think we need to expose that term to curriculum writers, and we only chose it to be different from "Course" due to the hassle of already having so many things in the code already named "course", including the "courses" table which probably won't get removed from redshift for some time. maybe in the future when there are fewer things named "course" we can change this to just be "Is a course".

regardless, I would suggest including Express and Pre-Express in this list, to help make it more clear when this should be checked.

Reply:
I think calling it a standalone course makes sense or single unit course. I agree not exposing CourseOffering to curriculum writers is a good plan